pkg/koordlet/metricsadvisor/devices/helper/sriov.go

```diff
@@ -0,0 +1,84 @@
+package helper
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+const (
+	configuredVfFile = "sriov_numvfs"
+)
+
+// SriovConfigured returns true if sriov_numvfs reads > 0 else false
+func SriovConfigured(addr string) bool {
+	return GetVConfigured(addr) > 0
+}
+
+func extractNumber(pfDir string, s string) int {
+	num, _ := strconv.Atoi(strings.TrimPrefix(s, fmt.Sprintf("%s/virtfn", pfDir)))
+	return num
+}
+
+// GetVFList returns a List containing PCI addr for all VF discovered in a given PF
+func GetVFList(pf string) (vfList []string, err error) {
+	vfList = make([]string, 0)
```

Comment:
Please mv the file operations under the directory pkg/koordlet/util/system first.
BTW, we should make the file operation cacheable to avoid unnecessary I/O especially when the file contents are unchanged. Please follow these two packages `pkg/koordlet/resourceexecutor` and `pkg/koordlet/util/system`. And if we do not have enough time to implement them, please leave TODOs.